packages/modular-scripts/src/test/utils.ts

```diff
@@ -53,3 +56,74 @@ export async function addFixturePackage(
     );
   }
 }
+
+/**
+ * Creates a temporary Modular repo with Modular node_modules symlinked
```

Comment:
Can you describe a bit better how this works + the motivations behind this for future memory? I expect people reading this code not understanding why we're doing this immediately.

Reply:
Yep, will add more details

Reply:
Added some detail, let me know if you think it's sufficient